### PR TITLE
content: publish schematic checklist

### PR DESCRIPTION
**Pull request by bahildebrand**

### CODE REVIEW

File: _posts/2024-07-02-schematic-review-checklist.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Here's my current checklist. I didn't list all the stories as above (this would'
 
 - [ ] How will you (a firmware engineer) attach a debugger to the board? (By soldering is not usually a good answer).[5](#story-5-too-much-or-to-little-debug)
 - [ ] Are non-essential, but helpful, GPIOs exposed for a debugger, such as SWO or ETM?
-- [ ] Do test points exist for power rails, analog signals, and busses?
+- [ ] Do test points exist for power rails, analog signals, and buses?
 
 
 ## Conclusion
```
Comment (gminn) [inline _posts/2024-07-02-schematic-review-checklist.md:112]: I don't want to downplay what you've provided -- we know from the intro this isn't an exhaustive list. Instead of `This is the start of my checklist for schematic reviews.`, how about `This checklist captures the most important items I have learned to ask about as a firmware developer in schematic reviews."`? @schultetwin

File: _posts/2024-07-02-schematic-review-checklist.md [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
 
 Some pre-schematic review checklist items:
 
- - [ ] Get invited to the schematic review. Really, this is a callout to be working with your electrical engineer. If you've been working with your electrical engineer, providing guidance on what hardware you'll need, being invited to the schematic review should be a given.
- - [ ] Build your checklist up front. Use this blog post as a starting point, but make sure you customize the list for the given project.
+ - [ ] Get invited to the schematic review. Really, this is a callout to be working with your electrical engineer. If you've been working with your electrical engineer, providing guidance on what hardware you'll need, being invited to the schematic review should be a given.
```
Comment (gminn) [inline _posts/2024-07-02-schematic-review-checklist.md:32]: Minor formatting - what do you think of checking the boxes with `[x]`? Gives a nice feeling of completion to me for some reason. Here is a comparison:

Comment (gminn) [reply _posts/2024-07-02-schematic-review-checklist.md:32]: Side note, I wish just the checkbox got rendered and not the bullet, but I think this is just how it is if we use checkboxes

**Review by gminn — CHANGES_REQUESTED** [review]
A few minor fixes and some questions for @schultetwin!

File: _posts/2024-07-02-schematic-review-checklist.md [hunk 3]
```diff
@@ -1,45 +1,45 @@
 ---
```
Comment (gminn) [inline _posts/2024-07-02-schematic-review-checklist.md:1]: Last couple things (making a comment to track them):
- update date to 7/17
- run prettier `npx prettier --prose-wrap always --write [post]` -- I've been running this manually lately but Noah has a PR up to add to it to the precommit hook. I would put this formatting in a separate commit since it will make changes hard to see.

**Comment by schultetwin** [comment]
The changes look great to me! Should I make the changes? Or should I have
y'all merge the PR and then make more?

Mark

**Comment by gminn** [comment]
> The changes look great to me! Should I make the changes? Or should I have y'all merge the PR and then make more? Mark

@schultetwin Nope we'll handle them!

**Review by gminn — APPROVED** [review]
